app/codes/signmanager.py

```diff
@@ -45,17 +45,20 @@ def sign_transaction(wallet_data, transaction_data):
 
 
 def sign_object(private_key, data):
-    pvtkeybytes = base64.b64decode(private_key)
+    pvtkeybytes = bytes.fromhex(private_key)
     msg = json.dumps(data).encode()
     sk = ecdsa.SigningKey.from_string(pvtkeybytes, curve=ecdsa.SECP256k1)
     msgsignbytes = sk.sign(msg)
-    msgsign = base64.b64encode(msgsignbytes).decode('utf-8')
+    msgsign = msgsignbytes.hex()
     return msgsign
 
-
-def verify_sign(data, signature, public_key):
-    public_key_bytes = base64.b64decode(public_key)
-    sign_trans_bytes = base64.decodebytes(signature.encode('utf-8'))
-    verifying_key = ecdsa.VerifyingKey.from_string(public_key_bytes, curve=ecdsa.SECP256k1)
+def verify_sign(data, public_key,signature):
```

Comment:
@Prannav-k this swap is expected?

Reply:
no , this seems a bug. it should be def verify_sign(data, signature, public_key)